metrics/wakumetrics/client.go

```diff
@@ -0,0 +1,183 @@
+package wakumetrics
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/status-im/status-go/protocol/transport"
+	wakutypes "github.com/status-im/status-go/waku/types"
+	"github.com/status-im/status-go/wakuv2"
+
+	v1protocol "github.com/status-im/status-go/protocol/v1"
+	v2common "github.com/status-im/status-go/wakuv2/common"
+	wps "github.com/waku-org/go-waku/waku/v2/peerstore"
+	v2protocol "github.com/waku-org/go-waku/waku/v2/protocol"
+)
+
+type ReceivedMessages struct {
```

Comment:
I suppose prometheus marshals these types to json?
Then we should explicitly define the json tags for each of these structs fields.

Reply:
These are just used for the function arguments in `PushReceivedMetrics`, the data pushed to prometheus is a numerical value and string labels. Each metric defines its own keys for the labels.